src/main/java/com/dougnoel/sentinel/filemanagers/FileManager.java

```diff
@@ -98,4 +104,78 @@ public static String getClassPath(String className) {
 			return null;
 		}
 	}
+	
+	/**
+	 * Saves an image File to an optionally set sub-directory of the configured, console set, or default "logs/images" directory.
+	 * Can optionally save to the root image directory if subDirectory is null.
+	 * 
+	 * @param subDirectory String the sub-directory to use
+	 * @param fileName String the file name of the image
+	 * @param imageFile File the File of the image to save
+	 */
+	public static void saveImage(String subDirectory, String fileName, File imageFile) throws IOException {
+		FileUtils.copyFile(imageFile, constructImagePath(subDirectory, fileName));
+	}
+	
+	/**
+	 * Saves a BufferImage to an optionally set sub-directory of the configured, console set, or default "logs/images" directory.
+	 * Can optionally save to the root image directory if subDirectory is null.
+	 * 
+	 * @param subDirectory String the sub-directory to use
+	 * @param fileName String the file name of the image
+	 * @param imageFile BufferdImage the BufferedImage to save
+	 */
+	public static void saveImage(String subDirectory, String fileName, BufferedImage imageFile) throws IOException {
+		File destinationFile = constructImagePath(subDirectory, fileName);
+		FileUtils.forceMkdir(destinationFile.getParentFile());
+		
+		ImageIO.write(imageFile, "png", destinationFile);
+	}
+
+	/**
+	 * Reads an image from disk from the configured, console set, or default "logs/images" directory.
+	 * Optionally can read from the root directory if subDirectory is null.
+	 * 
+	 * @param subDirectory String the sub-directory to use
+	 * @param fileName String the file name of the image
+	 * 
+	 * @return BufferedImage the image file read from disk
+	 */
+	public static BufferedImage readImage(String subDirectory, String fileName) {
+		return ImageComparisonUtil.readImageFromResources(constructImagePath(subDirectory, fileName).getAbsolutePath());
+	}
+	
+	 /**
+	 * Returns a String of the directory to use for test images set in the config file, command line,
+	 * or alternatively "logs/images" by default.
+	 * 
+	 * @return String the configured, console set, or default directory if none is found
+	 */
+	public static String getImageDirectory() {
+		String imageDirectory = Configuration.toString("imageDirectory");
+	    if(imageDirectory == null) {
+	    	imageDirectory = "logs/images";
+		}
+	    
+	    return imageDirectory;
+	}
+    
+	/**
+	* Returns a File use for saving an image using an optional sub-directory,
+	* file name, and the configured, console set, or default "logs/images" directory.
+	* Will use the root directory if subDirectory is null.
+	* 
+	* @param subDirectory String the sub-directory to use
+	* @param fileName String the file name of the image
+	* 
+	* @return File the constructed image File from directories and filename
+	*/
+	private static File constructImagePath(String subDirectory, String fileName) {
+		if(subDirectory == null) 
+			subDirectory = "";
+		else
+			subDirectory += "/";
```

Comment:
3 tests are failing on my Mac. I think it's related to this. We need to look at it together,

Reply:
What's the error in particular? I don't have a Mac to reproduce on. It's likely to do with how blank directories to use root are being put together.

Reply:
Switched to using File.separator and converting separators to it. Lemme know how it flies.

Reply:
Fixed